js/demo-app/public/components/example1/index.html

```diff
@@ -8,12 +8,14 @@ <h1>Component #1</h1>
     <li><a href="#/two/second">/two/second</a></li>
 </ul>
 
+<button class="toast" data-component-name="Component #1">Send Toast</button>
```

Comment:
@MattCheely I can easily add another tab to the nav and a more full-featured demo for the toasts.  It could include title, message, and level using form fields.  Thoughts?  Sufficient for now?